api/Element.json

```diff
@@ -1663,8 +1666,12 @@
               "version_added": "9",
               "notes": "Before Internet Explorer 9, this event is not supported via <code>addEventListener</code>; however, the event handler is supported since IE 5.5. The event can be listened to via <code>element.oncopy</code>."
             },
-            "opera": "mirror",
-            "opera_android": "mirror",
+            "opera": {
+              "version_added": "≤12.1"
```

Comment:
I tested http://mdn-bcd-collector.appspot.com/tests/api/HTMLElement/copy_event in Opera 12.16 for this.